public/client.css

```diff
@@ -0,0 +1,75 @@
+body {
+  background-color: #FFF;
+  font-family: 'FiraSans', sans-serif;
+}
+.generic-loader-wrap {
+  left: calc(50% - 50px);
+  position: relative;
+  margin-top: 15%;
+  width:100px;
+  height: 100px;
+  border-radius: 50%;
+}
+.generic-loader-wrap:before {
+  content: url(http://hoodiehq.github.io/hoodie-css/src/content_img/animals/low-profile-dog-1.png);
```

Comment:
once we use inert (see comment below), let’s put this image into the `public/` folder and load it with a relative path. We try avoiding loading assets from URLs to make sure that the apps work when run locally, even when not connected to the internet